### ISSUE TITLE
Skipping Front matter

### ISSUE THREAD

**Issue by woneob** [sasstools/sass-lint#542]
Some static site generater such as [Jekyll](http://jekyllrb.com/) include the [YAML front matter block](http://jekyllrb.com/docs/frontmatter/) at the top of the scss file for their own tracking purposes. [Front matter is necessary to compile](http://jekyllrb.com/docs/assets/).

```css
---
layout: null
someProp: value
---

html {
    background-color: #fff;
}
```

However, syntax error occurs in the sass-lint because the line of triple dashes `---`. 
Here is sass-lint's error log (I'm using [SublimeLinter-contrib-sass-lint](https://github.com/skovhus/SublimeLinter-contrib-sass-lint)). And linter does not work anymore.
```
c:/some/path/style.scss
  1:1  error  Please check validity of the block starting from line #1  Fatal

✖ 1 problem (1 error, 0 warnings) 
```

Can anyone help how to solve this issue?
How about add an option for skipping front matter. see [haml-lin#skipping-frontmattert](https://github.com/brigade/haml-lint#skipping-frontmatter)

---

See also:
- [Setting up sass with Jekyll](http://blog.davepoon.net/2015/01/19/setting-up-sass-with-jekyll/)

**Comment by DanPurdy** [comment]
You'll need to raise the issue with our AST provider `gonzales-pe` Unfortunately we wouldn't have control over skipping this in a friendly way as the AST needs to parse the whole file first for us to know the content of it.

You can file an issue [here](https://github.com/tonyganch/gonzales-pe/issues)

**Comment by rugk** [comment]
I've done it: https://github.com/tonyganch/gonzales-pe/issues/195

**Comment by DanPurdy** [comment]
Reopened to continue discussion around this.
